Updated GCP Terraform module's Installation method (#1519)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/gcp/installation.md

:::

```bash
echo 'provider "google-beta" { user_project_override = true }
provider "google" { user_project_override = true }
module "gcp" {
source = "port-labs/integration-factory/ocean//examples/gcp_cloud_run" 
version = ">=0.0.31" 
port_client_id = "<your_port_client_id>" 
port_client_secret = "<your_port_client_secret>" 
gcp_ocean_integration_image = "<your_artifact_registry_/_dockerhub>/port-ocean-gcp:<your_version>" 
gcp_organization = "<your_gcp_organization>" 
gcp_ocean_setup_project = "<your_gcp_project>" 
gcp_included_projects = ["<your_gcp_project>"] # The Project list that the integration digests resources from.
integration_identifier = "gcp"
scheduled_resync_interval = 1440
event_listener = {
  type = "POLLING"
}
}' > main.tf
# Initializing Terraform and Providers
terraform init
# Creating the resources (Cloud Run, PubSub Topic + Subscription, Cloud Assets Feed, Service account + Role)
terraform apply
```

> <h2> Our module's configuration options </h2>

Reviewer: ```suggestion
<h2>Configuration options</h2>
```

> Our GCP Ocean integration's Terraform module offers a set of configurations:

Reviewer: ```suggestion
The Port GCP integration's Terraform module offers a set of configurations:
```
